src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -1112,28 +1354,32 @@ private void AddMetaKeyValues(int i, DataViewSchema.Annotations.Builder builder)
 
             private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, string srcVariable, string dstVariable)
             {
-                string opType;
                 string castOutput;
                 OnnxNode castNode;
                 OnnxNode murmurNode;
 
-                opType = "MurmurHash3";
-                string murmurOutput = ctx.AddIntermediateVariable(_dstTypes[iinfo], "MurmurOutput");
-                var srcType = _srcTypes[iinfo].GetItemType().RawType;
-                if (_srcTypes[iinfo] is KeyDataViewType)
+                var srcType = _srcTypes[iinfo].GetItemType();
+                if (srcType is KeyDataViewType)
+                    return false;
+                if (_parent._columns[iinfo].Combine)
                     return false;
```

Comment:
@lynx1820 Any thoughts why we are not supporting key types?

Reply:
Thanks for the remainder! I'll work on adding support for keytypes. Zero value keys don't get mapped to a hash but map to zero, a behavior not currently present.

Reply:
As @lynx1820 mentioned, in key types 0 is mapped to 0 instead of to its hash value, so it would be more complicated to export to ONNX.

---
In reply to: [424584542](https://github.com/dotnet/machinelearning/pull/4828#discussion_r424584542) [](ancestors = 424584542)